src/js/components/AppListTypeFilterComponent.jsx

```diff
@@ -4,6 +4,7 @@ var AppTypes = require("../constants/AppTypes");
 
 var AppListTypeFilterComponent = React.createClass({
   displayName: "AppListTypeFilterComponent",
+  types: [],
```

Comment:
This is not needed and a React anti-pattern.

If you need such variable you could define a _private_ variable outside the react-class definition, but be aware that this is kind of static and shares the same data to duplicate components.

Alternatively if you need to access data on the component, you could define statics on the component like so:
```
static: {
 types: ["bla", "bla"]
} 
```
But beware this is also static data.

If you work with dynamic data, you basically only have 3 options.
1. Getting it through the component-properties somehow from the outside.
2. Getting it from a flux-store. Setting it through an action.
3. The real react way -> -modify the state-. This triggers a rerendering of course. If you don't want to rerender then this is a sign that you probably do something wrong.